cdap-examples/StreamConversion/src/main/java/co/cask/cdap/examples/streamconversion/StreamConversionApp.java

```diff
@@ -22,7 +22,6 @@
 import co.cask.cdap.api.data.stream.Stream;
 import co.cask.cdap.api.dataset.lib.FileSetProperties;
 import co.cask.cdap.api.dataset.lib.TimePartitionedFileSet;
-import co.cask.cdap.api.schedule.Schedules;
```

Comment:
It doesn't, since I added a new import but forgot to remove this one and update line number in a previous PR. Line number stays the same now.